source/_includes/installation/operating_system.md

```diff
@@ -157,6 +157,23 @@ _Select and copy the URL or use the "copy" button that appear when you hover it.
 3. Power the system on.
 
    - Wait for the Home Assistant welcome banner to show up in the console of the generic-x86-64 system.
+
+<div class="note">
+If the machine complains about not being able to find a bootable medium, you might need to specify the EFI entry in your BIOS.
+This can be accomplished either by using a live operating system (e.g. Ubuntu) and running the following command:
+
+  ```text
+  efibootmgr --create --disk /dev/sda --part 1 --label "HAOS" --loader \EFI\BOOT\bootx64.efi
```

Comment:
I'd prefer adding a newline here so it looks nicer in the docs, e.g.

```
efibootmgr --create --disk /dev/sda --part 1 --label "HAOS" \
    --loader \EFI\BOOT\bootx64.efi
```

Also, can you replace `sda` with a placeholder and note that it should point to the drive where Home Assistant OS has been installed to, typically `sda` or `nvme0n1` if a NVMe SSD is used.